package-lock.json

```diff
@@ -1706,7 +1714,7 @@
         },
         "minimist": {
           "version": "1.2.0",
-          "resolved": "https://registry.npmjs.org/minimist/-/minimist-1.2.0.tgz",
+          "resolved": "http://registry.npmjs.org/minimist/-/minimist-1.2.0.tgz",
```

Comment:
These kind of changes shouldn't happen with the latest Node.js >= 8.x.

Reply:
```suggestion
          "resolved": "http://registry.npmjs.org/minimist/-/minimist-1.2.0.tgz",
```